pkgs/by-name/cl/cli-tips/package.nix

```diff
@@ -0,0 +1,51 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  bash,
+  makeWrapper,
+  nix-update-script,
+}:
+
+stdenvNoCC.mkDerivation {
+  pname = "cli-tips";
+  version = "0-unstable-2024-11-14";
+
+  src = fetchFromGitHub {
+    owner = "cli-stuff";
+    repo = "cli-tips";
+    rev = "ebc191a54be7e39accd1948c3de8aded438d0495";
+    sha256 = "sha256-KELgatdL+2M5ktuAHEljIEJ9wqP578dp5tYWYPpP3bg=";
+  };
+
+  buildInputs = [ bash ];
```

Comment:
Is this really necessary? `stdenv` already has bash anyway

Reply:
I used neofetch as a template